Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
> Apple was explicitly declining apps that allowed signup...

I think they've changed their stance now... They *want* signup as long as there is no upgrade option that doesn't go via In-app purchases or subscriptions.